src/sensors/rev_encoder.hpp

```diff
@@ -1,57 +1,33 @@
-/**
- * @file rev_encoder.hpp
- * @brief The header file for the Rev Encoder firmware
- * @date 2023-10-31
- */
-
 #include <cmath>
 #include <FreqMeasureMulti.h>
 
 #ifndef REV_ENCODER_H
 #define REV_ENCODER_H
 
-/**
- * @brief the class for the Rev Encoder
- */
+/// @brief the class for the Rev Through Bore Encoder(www.revrobotics.com/rev-11-1271/)
 class RevEncoder 
 {
 	private:
-		/**
-		* @brief the pin number that the encoders signal pin is plugged into
-		*/
+		/// @brief the pin number that the encoders signal pin is plugged into
 		uint8_t inPin;
-		/**
-		* @brief Used to read rise time of the encoder
-		*/
+		/// @brief Used to read rise time of the encoder
 		FreqMeasureMulti freq;
-		/**
-		* @brief measure of current angle in ticks (1024 ticks = 1 rotation). When set to 1 individual readings are returned
-		*/
+		//// @brief measure of current angle in ticks (1024 ticks = 1 rotation). When set to 1 individual readings are returned
 		float ticks;
-		/**
-		* @brief measure of current angle in radians 
-		*/
+		/// @brief measure of current angle in radians 
 		float radians;
 	public:
-		/**
-		* @brief Construct a new rev_encoder object
-		* @param encPin the pin number that the encoders signal pin is plugged into
-		* @param baudrate the desired baudrate for the encoder
-		*/
+		/// @brief Construct a new rev_encoder object
+		/// @param encPin the pin number that the encoders signal pin is plugged into
```

Comment:
encPin should be snake_case (also I suggest not shortening variable names if you can help it, I.e. encoder_pin instead)